src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -1898,7 +1895,7 @@ public static function createNote(array $item, bool $api_mode = false): array
 		 * }
 		 */
 
-		if (empty($item['uid']) || !Feature::isEnabled($item['uid'], Feature::EXPLICIT_MENTIONS)) {
+		 if (empty($item['uid']) || !Feature::isEnabled($item['uid'], Feature::EXPLICIT_MENTIONS)) {
```

Comment:
Superfluous whitespace.
```suggestion
		if (empty($item['uid']) || !Feature::isEnabled($item['uid'], Feature::EXPLICIT_MENTIONS)) {
```

Reply:
Done